src/components/Logs.tsx

```diff
@@ -45,17 +77,22 @@ function Logs({ token, defaultMessage }: Props) {
     );
 
     return (
-        <LazyLog
-            extraLines={1}
-            stream={true}
-            text={logs
+        <Editor
+            height={`${height ?? 200}px`}
+            defaultLanguage=""
+            theme={theme.palette.mode === 'light' ? 'vs' : 'vs-dark'}
+            options={{
+                lineNumbers: 'off',
+                readOnly: true,
+                scrollBeyondLastLine: false,
+            }}
```

Comment:
Eventually we might need a wrapper around the editor that provides the basic default options we always want.